src/map/packets/basic.h

```diff
@@ -29,7 +29,8 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include <stdio.h>
 #include <string.h>
 
-#define PACKET_SIZE 0x104
+// Max packet size
+#define PACKET_SIZE 0x1FF
```

Comment:
Is this true across the board? This means every packet we allocate will be bigger. Just wanted to scrutinise a bit

Reply:
yes, client can accept up to 0x1FF bytes

Reply:
This might be better renamed as MAX_PACKET_SIZE, provided we actually truncate these based on size and aren't always sending out 0x1FF sized packets. As always, maybe a comment explaining this to passers-by, and those of us who forget things

Reply:
GetSize() is called in map.cpp to determine how much of the buffer to copy already, it was just named this way the whole time